### PR TITLE
Bug 1946079: Modify dhcp kernel args based on ipv4/ipv6

### PR DESCRIPTION
**Pull request by sadasu**

### CODE REVIEW

**Comment by asalkeld** [comment]
/retest
cbo PR has merged

File: get-resource.sh [hunk 1]
```diff
@@ -63,6 +63,11 @@ else
       unxz "$RHCOS_IMAGE_FILENAME_RAW"
     fi
 
+    if [ -n "$IP_OPTIONS" ] ; then
```
Comment (hardys) [inline get-resource.sh:66]: This could be undefined, so it'd be safer to do `${IP_OPTIONS:-}` or else default this at the top of the script?

**Pull request closed without merging**